site/en/docs/extensions/reference/declarativeNetRequest/index.md

```diff
@@ -78,18 +88,18 @@ the request URL. Some examples of URL filters:
 
 ## Dynamic rules
 
-To add or remove rules dynamically, extensions can use the [updateDynamicRules][9] API method.
+To add or remove rules dynamically, extensions can use the [updateDynamicRules][12] API method.
```

Comment:
s/extensions can//

To add or remove rules dynamically, use the...

Reply:
thx